index.js

```diff
@@ -101,6 +114,21 @@ const ignored = [
     property: "position",
     value: "fixed",
     ignoredProperties: ["float", "clear", "vertical-align"]
+  },
+  {
+    property: "flex-basis",
+    value: "/^.*$/",
+    ignoredProperties: ["width", "height"]
+  },
+  {
+    property: "list-style-type",
```

Comment:
> list-style-image has no effect on this element since it is list-style-type:none

https://github.com/captainbrosset/useless-css-properties/blob/master/rules.md#misc